modules/sfp_zetalytics.py

```diff
@@ -60,13 +63,28 @@ def watchedEvents(self):
         return ["INTERNET_NAME", "DOMAIN_NAME", "EMAILADDR"]
 
     def producedEvents(self):
-        return ["INTERNET_NAME", "AFFILIATE_DOMAIN_NAME"]
+        return ["INTERNET_NAME", "AFFILIATE_DOMAIN_NAME", "INTERNET_NAME_UNRESOLVED"]
 
     def emit(self, etype, data, pevent):
+        if self.checkForStop():
+            raise ModuleStop()
```

Comment:
Please stick with the existing approach for now. We will be revisiting the approach to this later on.